guides/doc-Release_Notes/topics/foreman.adoc

```diff
@@ -15,4 +15,7 @@ There are no upgrade warnings with Foreman {ProjectVersion}.
 [id="foreman-deprecations"]
 == Deprecations
 
-There are no deprecations with Foreman {ProjectVersion}.
+=== Compute resources - {oVirt}
```

Comment:
We usually refer to the RFC so please include a link to https://community.theforeman.org/t/rfc-deprecation-and-removal-of-ovirt-compute-resource/42527.